lmdeploy/serve/openai/api_server.py

```diff
@@ -1256,6 +1263,8 @@ def serve(model_path: str,
 
     handle_torchrun()
     _, pipeline_class = get_task(model_path)
+    if isinstance(backend_config, PytorchEngineConfig):
```

Comment:
It is noticed that `enable_mp_engine` defaults to False, but api_server changes it to True by force. So, we don't recommend it when pipeline is used or pipeline hasn't support this feature yet